test/fixtures/config/.soycriticrc

```diff
@@ -1,4 +1,8 @@
 {
   "callToImportRegex": "(\\S+)",
-  "callToImportReplace": "{$1|param}"
+  "callToImportReplace": "{$1|param}",
+  "implicitParams": {
+    "*Component": ["additionalParam"],
+    "Foo": ["bar"]
```

Comment:
It's fine, we can use the first regex in `implicitParams` as an expected failing test case, eventually.